src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      action="http://localhost:8080/"
+      method="post"
+    >
+      <label class="label">
+        <input
+          class="search"
+          type="text"
+          data-qa="big keypress"
```

Comment:
add attribute data-qa="big" for big search form, and data-qa="small" for small

```suggestion
          data-qa="keypress"
```